### ISSUE TITLE
Schvalování uživatelských žádostí pokud je dílo součástí PXN

### ISSUE THREAD

**Issue by CIZKOVAL** [NLCR/SeznamDNNT#413]
Dobrý den,
pokud je dílo současně v žádosti PXN (pravděpodobně volné dílo) a zároveň je v nějaké uživatelské žádosti (VN, VNZ, VNL, NZN), pak by mělo mít dílo **příznak PX** a řešení by mělo být následující:

**Schválit uživatelskou žádost nepůjde dokud nebude dílo schválené/zamítnuté v PXN žádosti.** Kurátor může uživatelskou žádost pouze zamítnout.
Když je dílo schválené/zamítnuté v PXN žádosti, tak dílo v uživatelské žádosti (VN, VNL, VNZ, NZN)  půjde schválit/zamítnout.

**Comment by CIZKOVAL** [comment]
Nyní nemá dílo v žádosti příznak PX a uživatelská žádost (např. VN) lze schválit a zároveň lze poté schválit i žádost PXN. V tomto konkrétním případě tedy dílo v žádosti VN získá stav N. A schválením žádosti PXN dostane stav X, to je špatně.

**Comment by Cypzirschova** [comment]
Pokud dostane dílo v žádosti PXN po schválení stav X, tak by se neměla dál žádost zpracovávat. Kurátor by jí měl zamítnout. Dílo, které je volné (X) v seznamu být nemá a ani se nemá dát znovu zařadit. Takže pokud bude dílo v jakékoliv žádosti (NZN, VN, VNL, VNZ) po updatu stavu (na X), žádost by měl kurátor zamítnout. Případně by se to asi dalo udělat automatizovaně.

**Comment by Cypzirschova** [comment]
Ještě bych doplnila, co jsem říkala na dnešní schůzce. Stav PX používáme i u děl, která nejsou zdigitalizována. Pokud bude mít dílo stav PX a **nebude součástí žádostí PXN**, mělo by se dát dál pracovat s dílem ve všech žádostech (schvalovat/zamítat dle potřeby). @Lichtenbergova

**Comment by CIZKOVAL** [comment]
Ano, tento [komentář](https://github.com/NLCR/SeznamDNNT/issues/413#issuecomment-1164290053) se týká issue, které budu terpve zakládat a propojím ho s tímto.

**Comment by CIZKOVAL** [comment]
Upravuji tedy body z úvodu následovně:

1. pokud je dílo v žádosti PXN (pravděpodobně volné dílo), pak má mít **kurátorský stav PX** (nyní ho nemá, pouze příznak PXN žádosti)
2. pokud je dílo současně v žádosti **PXN** (pravděpodobně volné dílo) **a zároveň** je **v** nějaké **uživatelské žádosti** (VN, VNZ, VNL, NZN), pak by řešení mělo být následující:
**Dokud nebude dílo schválené/zamítnuté v PXN žádosti, tak v uživatelské žádosti nepůjde schvalovat, pouze zamítat.**
**Pokud bude dílo v PXN žádosti schválené**, pak **v uživatelské žádosti** už nepůjde schvalovat, **pouze zamítat.**
**Pokud bude dílo v PXN žádosti zamítnuté**, pak **v uživatelské žádosti** půjde **jak schvalovat, tak zamítat.**

**Comment by pavel-stastny** [comment]
Bude řešeno dalším rozvojem.

**Comment by pavel-stastny** [comment]
> pokud je dílo v žádosti PXN (pravděpodobně volné dílo), pak má mít kurátorský stav PX (nyní ho nemá, pouze příznak PXN žádosti)

PXN stav v žádostech lze řešit konfigurací procesu v sekci `results`. 

> pokud je dílo současně v žádosti PXN (pravděpodobně volné dílo) a zároveň je v nějaké uživatelské žádosti (VN, VNZ, VNL, NZN), pak by řešení mělo být následující:
> ...
> Jde o stejný princip jako v DXN žádostech.

Myslím, že toto je na diskuzi.
